src/libsync/propagatorjobs.cpp

```diff
@@ -149,13 +149,16 @@ void PropagateLocalMkdir::startLocalMkdir()
     QDir newDir(propagator()->fullLocalPath(_item->_file));
     QString newDirStr = QDir::toNativeSeparators(newDir.path());
 
+    qCInfo(lcPropagateLocalMkdir) << "Starting Local Mkdir for:" << newDirStr;
```

Comment:
@mgallien Warning is for cases when something went "not exactly right", why would you use warning instead of info here? Other similar logs that mark the beginning of some method we have already, are all using info.  Are you sure warning should be used instead?

Reply:
true, I meant debug here, sorry